### ISSUE TITLE
typo in shutdown --help output

### ISSUE THREAD

**Issue by wassupluke** [systemd/systemd#31188]
### systemd version the issue has been seen with

255.3-1

### Used distribution

Arch Linux x86_64

### Linux kernel version used

6.7.2-arch1-2

### CPU architectures issue was seen on

x86_64

### Component

other

### Expected behaviour you didn't see

This is a compatibility interface, please use the more powerful 'systemctl _halt_', 'systemctl poweroff', 'systemctl reboot' commands instead.

### Unexpected behaviour you saw

This is a compatibility interface, please use the more powerful 'systemctl _reboot_', 'systemctl poweroff', 'systemctl _reboot_' commands instead.

### Steps to reproduce the problem

$ shutdown --help

### Additional program output to the terminal or log subsystem illustrating the issue

```sh
$ shutdown --help
shutdown [OPTIONS...] [TIME] [WALL...]

Shut down the system.

Options:
     --help      Show this help
  -H --halt      Halt the machine
  -P --poweroff  Power-off the machine
  -r --reboot    Reboot the machine
  -h             Equivalent to --poweroff, overridden by --halt
  -k             Don't halt/power-off/reboot, just send warnings
     --no-wall   Don't send wall message before halt/power-off/reboot
  -c             Cancel a pending shutdown
     --show      Show pending shutdown

This is a compatibility interface, please use the more powerful 'systemctl reboot',
'systemctl poweroff', 'systemctl reboot' commands instead.

See the shutdown(8) man page for details.
```

**Comment by wassupluke** [comment]
The help text cites 'systemctl reboot' twice, could perhaps be more helpful / less redundant to replace one of those instances with 'systemctl halt'.

**Comment by shuLhan** [comment]
I believe this has has been fixed in https://github.com/systemd/systemd/pull/31055 .

**Comment by yuwata** [comment]
Yes, it is already fixed by 3af9d86eb7f2030cb6f7a207cfbad7eec23631ca.
